stories/Autosuggest.js

```diff
@@ -75,7 +75,8 @@ storiesOf('Organisms|Autosuggest', module)
         };
 
         const getSelectedPlaceholder = () => {
-            const numOfItems = store.get('selectedSuggestions').length;
+            const items = store.get('selectedSuggestions');
+            const numOfItems = items ? items.length : 0;
             if (!numOfItems) {
```

Comment:
maybe it would be better to change the condition to `numOfItems === 0` just to be more transparent and consistent with the next one?